### DISCUSSION TITLE
Compatible with Intex Bubble Spa V26310 (bought 2022) - Thank you

### DISCUSSION THREAD

**Discussion by drheck** [Yogui79/IntexPureSpa#67] [General]
The cloud solution is terrible....

**Comment by LEECHER1** [comment]
we do not know it.  we don't have version 2022 but we think it works but
with probably changes in the code.

You can try it.

Dr. Peter Heck ***@***.***> schrieb am Mi., 31. Mai 2023,
17:30:

**Comment by drheck** [comment]
Thank you. Okay. I have to buy the LC21 chip......
